account/src/main/java/com/whz/account/resource/CommentToneSubscriber.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2020-2021, Fachgruppe Informatik WHZ <help.flaxel@gmail.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.whz.account.resource;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.DeliverCallback;
+import com.whz.account.infrastructure.CommentToneData;
+import com.whz.account.model.account.Account;
+import com.whz.account.model.account.AccountEntity;
+import io.vlingo.actors.Logger;
+import io.vlingo.actors.Stage;
+import io.vlingo.common.serialization.JsonSerialization;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * The CommentToneSubscriber which subscribes to a CommentTonePublisher endpoint and receives
+ * messages containing a comment tone. After receiving a message the "sentimentReceived(Sentiment
+ * sentiment)" method will be triggered which matches the id of the comment tone to our account and
+ * changes the given sentiment to the new value by creating a SentimentReceived event.
+ *
+ * @author Lation
+ */
```

Comment:
Personally, I would not write too many details in the class documentation. This should either be part of the domain story or explained through the methods. Otherwise, the class documentation will eventually become gigantic, as the complexity of the class increases.